docs/source/technical_tutorials/working_with_r/access_cmr_r.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>you should probably warn people about this throwing an error if it already exists</p><p>&gt;<code>dir.create</code>&nbsp;indicates failure if the directory already exists</p><p> </p><p>OR </p>

<pre class="ql-syntax" spellcheck="false">dir_name = "./data"
if(!dir.exists(dir_name)){ dir.create(dir_name)}
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/482/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Maybe just use <code>tempdir()</code> for this so users don't need to worry about deleting files either.</p>

Reply:
:thinking: for DPS jobs that makes sense, but for regular classic I want to keep and inspect?
`tempdir()` feels like a best practices topic - separate from this.